docs/guides/classical-feedforward-and-control-flow.ipynb

```diff
@@ -202,196 +192,218 @@
+    "\n",
+    "- Due to the limited physical memory on control electronics, there is also a limit on the number of `if` statements and size of their operands. This limit is a function of the number of broadcasts and the number of broadcasted bit in a job (not a circuit).\n",
+    "\n",
+    "   When processing an `if` condition, measurement data needs to be transferred to the control logic to make that evaluation. A broadcast is a transfer of unique classical data, and broadcasted bits is the number of classical bits being transferred. Consider the following:\n",
+    "\n",
+    "   ```python\n",
+    "   c0 = ClassicalRegister(3)\n",
+    "   c1 = ClassicalRegister(5)\n",
+    "   ...\n",
+    "   with circuit.if_test((c0, 1)) ...\n",
+    "   with circuit.if_test((c0, 3)) ...\n",
+    "   with circuit.if_test((c1[2], 1)) ...\n",
+    "   ```\n",
+    "   In the previous code example, the first two `if_test`s on `c0` are considered one broadcast because the content of `c0` did not change, and thus does not need to be re-broadcasted. The `if_test` on `c1` is a second broadcast. The first one broadcasted all three bits in `c0` and the second broadcasted just one bit, making a total of four broadcasted bits.\n",
+    "\n",
+    "   Currently, if you broadcast 60 bits each time, then the job can have approximately 300 broadcasts. If you broadcast just one bit each time, however, then the job can have 2400 broadcasts.\n",
+    "\n",
+    "- Arithmetic operations are not supported.\n",
+    "- Nested conditionals are not allowed. For example, the following code block will not work because it has an `if_test` inside another `if_test`:\n",
+    "   <Tabs>\n",
+    "    <TabItem value=\"Invalid\" label=\"Invalid\">\n",
+    "        ```python\n",
+    "           c1 = ClassicalRegister(1, \"c1\")\n",
+    "           c2 = ClassicalRegister(2, \"c2\")\n",
+    "           ...\n",
+    "           with circ.if_test((c1, 1)):\n",
+    "            with circ.if_test(c2, 1)):\n",
+    "             ...\n",
+    "        ```\n",
+    "     </TabItem>\n",
+    "     <TabItem value=\"Valid\" label=\"Valid\">\n",
+    "        ```python\n",
+    "        cr = ClassicalRegister(2)\n",
+    "        ...\n",
+    "        with circuit.if_test((cr, 0b11)):\n",
+    "          ...\n",
+    "        ```\n",
+    "     </TabItem>\n",
+    "    </Tabs>\n",
+    "\n",
+    "- Having `reset` inside conditionals is not supported.\n",
+    "- Limitations on classical registers:\n",
+    "      - The operand used in an `if_test` statement must be 32 bits or less. Thus, if you are comparing an entire `ClassicalRegister`, the size of that `ClassicalRegister` must be 32 bits or less. If you are comparing just a single bit from a `ClassicalRegister`, however, that `ClassicalRegister` can be of any size (since the operand is only 1 bit).\n",
```

Comment:
```suggestion
    "      - The operand used in an `if_test` statement must be 32 or fewer bits. Thus, if you are comparing an entire `ClassicalRegister`, the size of that `ClassicalRegister` must be 32 or fewer bits. If you are comparing just a single bit from a `ClassicalRegister`, however, that `ClassicalRegister` can be of any size (since the operand is only one bit).\n",
```